docs/blueprints/EP031.rst

```diff
@@ -259,13 +260,10 @@ V. Events
 VI. REST API
 =============
 
-  - POST /telemetry_int/v1/evc/ body evc_ids: [] for bulk insertions, if empty, then enable all. If invalid or non-existing EVC_ID are provided, abort the entire operation with 4XX status code.
-  - POST /telemetry_int/v1/evc/<evc_id>: enable/create INT flows for an EVC_ID.
-  - DELETE /telemetry_int/v1/evc/ body evc_ids: [] for bulk removals, if empty, then remove all. If invalid or non-existing EVC_ID are provided, abort the entire operation with 4XX status code.
-  - DELETE /telemetry_int/v1/evc/<evc_id>: disable/remove INT flows for an EVC_ID.
-  - GET /telemetry_int/v1/evc list all INT-enabled EVCs.
-  - POST /telemetry_int/v1/consistency/ body evc_ids: []- Force the consistency routine to run for evc_id's provided. If none are provide, force for all EVCs.
-
+  - ``POST /telemetry_int/v1/evc/enable`` body evc_ids: [] for bulk insertions, if empty, then enable all. If invalid or non-existing EVC_ID are provided, abort the entire operation with 4XX status code.
+  - ``POST /telemetry_int/v1/evc/disable`` body evc_ids: [] for bulk removals, if empty, then remove all. If invalid or non-existing EVC_ID are provided, abort the entire operation with 4XX status code.
+  - ``GET /telemetry_int/v1/evc`` list all INT-enabled EVCs.
+  - ``GET /telemetry_int/v1/evc_compare`` list and compare which telemetry_int flows are still coherent with EVC metadata status
```

Comment:
Right. Yes, thinking about it again, probably a `PATCH /telemetry_int/v1/evc/redeploy body evc_ids: []` would be handy, otherwise as you said you'd have to `/disable force` and then `/enable force`. I'll update the doc shortly, let me know if you'd be OK with it, it'd behave the same a `mef_eline` `redeploy`

Reply:
Included here 2fdcbdb7909c480383218ef8a4cb2f81f31aaec1